mozdownload/scraper.py

```diff
@@ -863,7 +862,6 @@ def platform_regex(self):
         PLATFORM_FRAGMENTS = {'linux': 'linux',
                               'linux64': 'linux64',
                               'mac': 'macosx',
-                              'mac64': 'macosx64',
```

Comment:
There is only one set of builds we have for Firefox. It's the bundle with 32bit/64bit binaries. The old bundle with PowerPC/32bit is not shipped anymore. So `mac` and `mac64` should point to the same bundle.

I may have written it the wrong way in issue #215. For consistency we should just keep mac64, just in case that we do not break any downstream projects.